conf/rockstor-collector.service

```
@@ -0,0 +1,19 @@
+[Unit]
+Description=Rockstor Data Collector
+After=network.target rockstor-pre.service rockstor.service
+Requires=rockstor-pre.service rockstor.service
```

```suggestion
Description=Rockstor Collector Service
After=rockstor.service
Requires=rockstor.service
```

Consistent use of service and capitalisation plus the single SGI dependency this service requires.